agent/consul/state/usage_oss.go

```diff
@@ -1,3 +1,4 @@
+//go:build !consulent
```

Comment:
Consul is still living in Go `1.16.7` so I don't think this needs to be here yet.

Reply:
we're running go `1.17` and vscode probably inserted it automatically. I can remove it if you prefer to leave it out but it shouldn't break anything should it?